package.json

```diff
@@ -7,13 +7,15 @@
     "lint": "next lint",
     "test": "jest --coverage",
     "cypress:open": "cypress open",
-    "cypress:run": "cypress run --record --key 083e76f3-5f44-43f0-b248-ace10abf0877"
+    "cypress:run": "cypress run --record --key 083e76f3-5f44-43f0-b248-ace10abf0877",
+    "format": "prettier --write ."
   },
   "dependencies": {
     "@auth0/auth0-react": "^1.10.1",
     "@headlessui/react": "^1.6.4",
     "@heroicons/react": "^2.0.10",
     "ace-builds": "^1.5.0",
+    "babel-plugin-transform-dynamic-import": "^2.1.0",
```

Comment:
What’s this for?

Reply:
I guess it’s for something, but I’m not sure why old tests that didn’t need it before now fail.